sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -887,20 +900,52 @@ def upload_manifest(
                 name=repository,
                 reference=tag_or_digest,
                 payload=data,
-                content_type=OCI_MANIFEST_MEDIA_TYPE,
-                headers={"Accept": OCI_MANIFEST_MEDIA_TYPE},
+                content_type=media_type,
```

Comment:
Should we still be passing the `Accept` header? Is it possible that the media type that I pass in is not going to be the media type I get back?

Reply:
- No, it's not necessary to pass `Accept`.
- Based on my tests, the _media type_ in `get_manifest` is always consistent with the _media type_ in `set_manifest`: i.e. https://github.com/Azure/azure-sdk-for-python/blob/bdeecb7a3c8a3d321745c4f7ad0fd91779fbddfc/sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client.py#L471